nemo/collections/tts/models/base.py

```diff
@@ -39,6 +41,18 @@ def generate_spectrogram(self, tokens: 'torch.tensor', **kwargs) -> 'torch.tenso
             sepctrograms
         """
 
+    @classmethod
+    def list_available_models(cls) -> 'List[PretrainedModelInfo]':
+        """
+        This method returns a list of pre-trained model which can be instantiated directly from NVIDIA's NGC cloud.
+        Returns:
+            List of available pre-trained models.
+        """
+        list_of_models = []
+        for subclass in cls.__subclasses__():
+            list_of_models.extend(subclass.list_available_models())
```

Comment:
This is fine here, but subclasses can potentially return None (that's the default), so it should be checked here before extend.

Reply:
That is a fair point, will update.